pom.xml

```diff
@@ -205,8 +205,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.1</version>
                 <configuration>
-                    <source>1.6</source>
-                    <target>1.6</target>
+                    <source>1.8</source>
+                    <target>1.8</target>
```

Comment:
The default method in the `MetricFilter` interface requires Java8.

For a fork that provides this functionality this is the simplest solution.

If the pull request would otherwise be accepted I can backport the code to Java6.